js/lib/alert-bar.js

```diff
@@ -0,0 +1,16 @@
+var $ = require('jquery')
+var callData = require('./communityCall.json')
+
+function callTime () {
+  var now = new Date().getHours()
+  var startTime = parseInt(callData.time, 10)
+  var dayOfWeek = new Date().getDay()
+  var reminder = $('.alert-bar')
+  var announcement = $('.lh-title').append('<span>The IPFS community call will start at ' + callData.time + '. Join us <a href=' + callData.zoomLink + '>here</a></span>')
+  if ((startTime - now <= 2 && startTime - now >= 0) && (dayOfWeek === 1)) {
```

Comment:
`dayOfWeek === 1` should be tied to the `day` property of the data in `communityCall.json` right? Otherwise changes to the day for a call will still cause this banner to be displayed on Monday.

Reply:
Good point. The call for the for seeable future will on Monday, but I really should be using the `day` property of `communityCall.json` which is better than just hard coding Monday.